metricbeat/module/kubernetes/container/container.go

```diff
@@ -110,6 +111,13 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) {
 		if err != nil {
 			m.Logger().Error(err)
 		}
+		// Enrich event with container ECS fields
+		containerEcsFields := ecsfields(event)
+		if len(containerEcsFields) != 0 {
+			e.RootFields = common.MapStr{
```

Comment:
This will override any existing `RootFields` right? `event` variable will already have `RootFields` coming from https://github.com/elastic/beats/blob/c1d4f28ff30e28429efed51c817cd02d46633aac/metricbeat/module/kubernetes/util/kubernetes.go#L495

Reply:
I added a check in 0a0a5439b407f0c1f99d0804344beeb68a737b16